Makefile

```diff
@@ -163,3 +163,6 @@ clean:
 .PHONY: watch
 watch:
 	while true; do inotifywait -qqr -e modify -e create -e delete src/ && $(MAKE); done
+
+run-dev:
+	crystal run -Dpreview_mt -Dexecution_context src/lavinmq.cr -- -D /tmp/amqp
```

Comment:
This is good, we have discussed adding something like this internally, but it should be in a separate PR with some explanation and possibly docs aswell :)

Reply:
I think we basically decided against having a run cmd in the Makefile and instead just updating the docs https://github.com/cloudamqp/lavinmq/pull/1091

Reply:
ok cool!

Reply:
Thank you guys. I'll remove from this PR